django/forms/fields.py

```diff
@@ -183,8 +183,11 @@ def widget_attrs(self, widget):
 
     def has_changed(self, initial, data):
         """
-        Return True if data differs from initial.
+        Return True if data differs from initial. Return False when field is
```

Comment:
"Always return False if the field is disabled since self.bound_data always uses the initial value in that case."